src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.bookshop.controller;
+
+import com.example.bookshop.dto.BookDto;
+import com.example.bookshop.dto.CreateBookRequestDto;
+import com.example.bookshop.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/books")
+public class BookController {
+    private final BookService bookService;
+
+    @GetMapping
+    public List<BookDto> getAll() {
+        return bookService.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.findById(id);
+    }
```

Comment:
I am not sure, but I guess if we need to create getAll and getId methods in BookController, it means that we need to create the same methods in Services and Repositories, because different naming referees us to different implementations. But better ask about it mentors and ignore this message, if I am not right xD

Reply:
It would be better to `get` entities from the service, but `find` entities from the repository.
It is not critical, but semantically it is more preferable.